docs/02-app/01-building-your-application/06-optimizing/10-third-parties.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Third Parties
+nav_title: Third Parties
+description: Optimize the performance of third-party libraries in your application with the `@next/third-parties` package.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+[**`@next/third-parties`**](/docs/app/api-reference/components/font) is a library that provides a
+collection of components and utilities that improve the performance and developer experience of
+loading popular third-party libraries in your Next.js application.
+
+> **Good to know**: `@next/third-parties` is a new **experimental** library that is still under
+> active development. We're currently working on adding more third-party integrations.
+
+All third-party integrations provided by `@next/third-parties` have been optimized for performance
+and ease of use.
+
+## Getting Started
+
+To get started, you must install the `@next/third-parties` library:
+
+```bash filename="Terminal"
+npm install @next/third-parties
+```
+
+## Google Third-Parties
+
+All supported third-party libraries from Google can be imported from `@next/third-parties/google`.
+
+### Google Maps Embed
+
+The `GoogleMapsEmbed` component can be used to add a Google Maps Embed to your page. By default, it
+uses the `loading` attribute to lazy-load the embed below the fold.
```

Comment:
We should add docs for what each of the attributes mean (e.g. for map mode, we should link to the Google Maps docs here: https://developers.google.com/maps/documentation/embed/embedding-map#choosing_map_modes)